epoxy-processor/src/main/java/com/airbnb/epoxy/ViewAttributeInfo.kt

```diff
@@ -128,6 +137,21 @@ internal class ViewAttributeInfo(
                 super.isRequired()
             }
 
+    private fun getViewAttributeType(
+            element: Element,
+            errorLogger: ErrorLogger
+    ): ViewAttributeType? {
+        return when {
+            element is ExecutableElement -> ViewAttributeType.Method
+            element.kind === ElementKind.FIELD -> ViewAttributeType.Field
```

Comment:
don't use triple equals here

Reply:
can you do `when(element.kind) is FIELD ... is METHOD...`?

Reply:
It couldn't use 'is'.  The compiler tells: "'is' over enum entry is not allowed, use comparison instead". I can use '==' instead of '===' but it actually doesn't make a difference for enum